contrib/python-mediafile/template.py

```diff
@@ -0,0 +1,23 @@
+pkgname = "python-mediafile"
+pkgver = "0.12.0"
+pkgrel = 0
+build_style = "python_pep517"
+hostmakedepends = [
+    "python",
+    "python-build",
+    "python-setuptools",
+    "python-wheel",
+]
+makedepends = ["python-flit_core"]
```

Comment:
same as above, this is in the wrong place and you then don't need setuptools/wheel